saw-core/src/Verifier/SAW/Simulator/Value.hs

```diff
@@ -371,6 +374,7 @@ suffixTValue tv =
     VSort {} -> Nothing
     VRecursorType{} -> Nothing
     VTyTerm{} -> Nothing
+    VStringType -> Nothing
```

Comment:
Similarly, I'm not sure about this use of `Nothing`. All of the other `Nothing` cases in this function also have corresponding `Nothing` cases in `asFirstOrderTypeTValue`, so I did the same for `VStringType` for consistency.